code/game/objects/items/weapons/stock_parts.dm

```diff
@@ -6,7 +6,14 @@
 	icon_state = "RPED"
 	item_state = "RPED"
 	w_class = WEIGHT_CLASS_HUGE
-	can_hold = list(/obj/item/stock_parts)
+	can_hold = list(
+		/obj/item/stock_parts,
+		// Added separately in Initialize to avoid picking up unwanted
+		// subtypes.
+		// /obj/item/reagent_containers/glass/beaker,
```

Comment:
```suggestion
		// Added separately in Initialize to avoid picking up unwanted subtypes.
```
Im.. still not entirely sure why you've commented out `.../glass/beaker` rather than just removing it, same with how the word `subtypes` is on its own line
If there's some reason that im missing, ignore this

Reply:
It's commented out so that it's visually in the list, since it *is* an accepted item, it's just not added there for technical reasons.  Adjusted the comment and left it on one line.